adapter/pkg/messaging/azure_connection.go

```diff
@@ -29,7 +29,7 @@ import (
 	logger "github.com/wso2/product-microgateway/adapter/internal/loggers"
 )
 
-var bindingKeys = []string{tokenRevocation, notification}
+var bindingKeys = []string{tokenRevocation, notification, stepQuotaThreshold, stepQuotaReset}
```

Comment:
Does the event handling part and data clean up would always be done by a single ASB cluster. If some day we are decided to use 2 different ASB deployments this will not work

Reply:
Yes. We are also publishing to same ASB.

Reply:
Agree on supporting topic subscriptions in a generic way. Created this issue to track. We will do the refactoring : https://github.com/wso2-enterprise/choreo/issues/7978